src/locales/fr.json

```
@@ -16,6 +16,11 @@
     "sign-out": "Déconnexion",
     "sign-in": "Connexion"
   },
+  "ui": {
+    "card": {
+      "register-here": "S’enregistrer"
```

This is not a regular apostrophe character
```suggestion
      "register-here": "S'enregistrer"
```